app/scripts/views/base.js

```diff
@@ -762,6 +762,13 @@ define(function (require, exports, module) {
       this._hasNavigated = true;
     },
 
+    navigateViaServer (url) {
```

Comment:
> I did see the name navigateToUrl elsewhere and I'm not a fan. To me it fails to convey any meaningful difference to navigate. When viewed alongside each other, those two seem completely ambiguous I think.

That method name isn't in the code currently, so must've been a comment.

> How about request vs navigate? Does that better indicate that one of them is going to make an HTTP request?

Just `request` is too generic, it could mean any request and doesn't imply any side effects.

`navigateAway`, `hardNavigate`, `leaveAppToUrl`, `redirectToUrl`, maybe even `setDocumentLocation`?